html-syntax/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>HTML Syntax</title>
+</head>
+
+<body>
+  <img src="https://yt3.ggpht.com/a/AGF-l78Ro1iteqHtMsi1w_L30T43wssUUm8JjTSuPw=s900-c-k-c0xffffffff-no-rj-mo" alt="Pikachu and Doritos">
+  <br>
```

Comment:
you want to avoid using `br` tags in your HTML. They should only be used for very specific text formatting for things like poetry.

Reply:
Noted with thanks.